i18n/en.json

```diff
@@ -4198,6 +4206,50 @@
     "id": "ent.cluster.stopping.info",
     "translation": "Cluster internode communication is stopping on %v with hostname=%v id=%v"
   },
+  {
+    "id": "ent.compliance.csv.attachment.export.appError",
+    "translation": "Unable to add attachment to the CSV export."
+  },
+  {
+    "id": "ent.compliance.csv.attachment.copy.appError",
+    "translation": "Unable to copy the attachment into the zip file."
+  },
+  {
+    "id": "ent.compliance.csv.file.creation.appError",
+    "translation": "Unable to create CSV export temporary file."
```

Comment:
Perhaps `Unable to create temporary CSV export file.` <-- assuming the "CSV export file" is temporary?

Reply:
Is not exactly that. We create a temporary zip file and we write the csv, the metadata.json, and all the attachments on it (to avoid to do it entirely in memory), and at the end, we copy this to the file storage (this is because currently, we aren't able to do this directly in storage without storing it anywhere before, memory or disk).

Reply:
Submitted a proposal (`Unable to create temporary CSV export file.`) based on other of your comments.